namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf

```diff
@@ -0,0 +1,72 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "example_team_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.1"
+  repo_name = "example-module"
+  team_name = "example-team"
+  /*
+    By default scan_on_push is set to true. When this is enabled then all images pushed to the repo are scanned for any security
+    / software vulnerabilities in your image and the results can be viewed in the console. For further details, please see: 
+    https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html 
+    To disable 'scan_on_push', set it to false as below:
+  scan_on_push = "false"  
+  */
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "example_team_ecr_credentials" {
+  metadata {
+    name      = "example-team-ecr-credentials-output"
+    namespace = "<NAMESPACE>"
```

Comment:
Please update your actual namespace name here. "hello-world-test"

Reply:
Apologies, this file was committed in error